.travis.yml

```diff
@@ -35,5 +55,6 @@ jobs:
 script:
   # race detector is only available on amd64
  - if [[ "$TRAVIS_CPU_ARCH" == "amd64" ]]; then go test -race -v ./...; else go test -v ./...; fi
+ - if [[ "$TRAVIS_CPU_ARCH" == "ppc64le" ]]; then go test -race -v ./...; else go test -v ./...; fi
```

Comment:
Could we meged this with the previous if ?

Reply:
yeah sure, Below one is fine?

if [[ "$TRAVIS_CPU_ARCH" == "amd64" ]] || [[ "$TRAVIS_CPU_ARCH" == "ppc64le" ]]; then go test -race -v ./...; else go test -v ./...; fi